### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SzymCrusader**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,48 @@
 # WattOhm
-Bank simulator for fun and games (AmperVolt in Rust)
+
+WattOhm is a bank simulator written in Rust
+
+it uses Rocket as a web framework
```
Comment (SaronTetra) [inline README.md:5]:
```suggestion
using Rocket as a web framework
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,48 @@
...
+and Diesel for operating on the database
+
+the database used is PostgreSQL
+
+* Rust
+  * Rocket
+  * Diesel
+* PostgreSQL
```
Comment (SaronTetra) [inline README.md:13]: Unnecessary repeat

File: README.md [hunk 3]
```diff
@@ -1,2 +1,48 @@
...
+## 
+Wattohm is meant to be used in games (e.g. Monopoly)
```
Comment (SaronTetra) [inline README.md:15]:
```suggestion
WattOhm can be used in games (e.g. Monopoly, tabletop RPG)
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,48 @@
...
+```
+start WattOhm
+```bash
+docker run wattohm -d
+```
+
+connect to Wattohm in web browser
+```
+host-ip:8000
+```
```
Comment (SaronTetra) [inline README.md:48]: Add a section on manual run (without docker)

**Review by SaronTetra — CHANGES_REQUESTED** [review]

**Comment by SzymCrusader** [comment]
Added requestet changes to README.md
Closing and opening ne pull request

**Pull request closed without merging**